test/test_nn.py

```diff
@@ -7056,11 +7056,12 @@ def test_pointwise_loss_broadcast(self):
     # https://github.com/pytorch/pytorch/issues/27692 reports
     # that l1_loss get a wrong result for big batch size
     def test_l1_loss_correct(self):
```

Comment:
Is there no other L1 loss test that should be updated? 

This is making me thing we should (eventually) create test_losses.py.